### PR TITLE
Resolver doc: how-to use with Vert.x Web Client

### PR DESCRIPTION
**Pull request by tsegismont**
Related to https://github.com/vert-x3/vertx-web/pull/2573

### CODE REVIEW

File: src/main/asciidoc/index.adoc [hunk 1]
```diff
@@ -2,22 +2,30 @@
...
+A service is addressed with a `{@link io.vertx.serviceresolver.ServiceAddress}` instead of a `{@link io.vertx.core.net.SocketAddress}`.
 
 [source,java]
 ----
 {@link examples.ServiceResolverExamples#usingHttpClient}
 ----
 
+[TIP]
+====
+To integrate Vert.x Service Resolver with the Vert.x Web Client:
```
Comment (vietj) [inline src/main/asciidoc/index.adoc:23]: I think we should get a section (even if it is copy/paste) "Getting started with the Vert.x Web client"

Comment (tsegismont) [reply src/main/asciidoc/index.adoc:23]: The problem is we can't get an example without introducing a dependency to vertx-web-client

Comment (vietj) [reply src/main/asciidoc/index.adoc:23]: we can work around with inline code without a class

Comment (tsegismont) [reply src/main/asciidoc/index.adoc]: Done

**Comment by tsegismont** [comment]
Note that the example provided for Web Client won't work until https://github.com/vert-x3/vertx-web/pull/2573 is merged.

**Comment by tsegismont** [comment]
@vietj ok to merge?

**Review by vietj — APPROVED** [review]

**Pull request merged**